README.md

```diff
@@ -16,35 +16,54 @@ The structure of this file is defined in [assets/schema_input.json](assets/schem
 
 # Parameters
 
-The main parameters are `--input` as defined above and `--output` for specifying the output results directory.
+## Mandatory
+
+The mandatory parameters are as follows:
+
+- `--input` : a URI to the samplesheet as specified in the [Input](#input) section.
+- `--output` : to specify the output results directory.
+- `--kraken2_db /path/to/kraken2database` : to specify the directory to the Kraken2 database
+- `--bracken_db /path/to/brackendatabase` : to specify the directory to the Bracken database
+
+## Optional
 
 Additionally, you may wish to provide:
 
-`-profile singularity` to specify the use of singularity containers
+### SpeciesAbundance Parameters
 
-`-r [branch]` to specify which GitHub branch you would like to run
+- `--taxonomic_level` : to specify the taxonomic level of the bracken abundance estimation.
+  - Must be one of 'S'(species)(default), 'G'(genus), 'O'(order), 'F'(family), 'P'(phylum), or 'K'(kingdom)
 
-`--kraken2_db /path/to/kraken2database`
+### Other Parameters
 
-`--bracken_db /path/to/brackendatabase`
+- `-profile` : to specify which profile to use (ex: `-profile singularity`)
+- `-r [branch]` : to specify which GitHub branch you would like to use
 
 Other parameters (defaults from nf-core) are defined in [nextflow_schema.json](nextflow_schmea.json).
 
 # Running
 
-To run the pipeline, please do:
+## Test Data
+
+To run the pipeline, please run:
 
 ```bash
-nextflow run phac-nml/speciesabundance -profile singularity -r dev -latest --input /path/to/samplesheet.csv --outdir results
+nextflow run phac-nml/speciesabundance -profile singularity -r dev -latest --input /path/to/samplesheet.csv --kraken2_db /path/to/kraken2_db -- bracken_db /path/to/bracken_db --outdir results
 ```
 
-Where the `samplesheet.csv` is structured as specified in the [Input](#input) section.
+The pipeline output will be written to a directory named `results`. A JSON file for integrating with IRIDA Next will be written to `results/iridanext.output.json.gz` (as detailed in the [Output](#output) section)
 
-# Output (in development)
+To run the pipeline using the test profile, please run:
+
+```bash
+nextflow run phac-nml/speciesabundance -profile docker,test -r dev -latest --outdir results
```

Comment:
Your call on this, but you may want to change `dev` to `main` here, or update on merging into main.